tests/integration/test_package_managers.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import logging
+import os
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        # Test scratch build with parent from external registries
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachito-gomod-with-deps.git",
+                ref="4c65d49cae6bfbada4d479b321d8c0109fa1aa97",
+                test_case="gomod_with_deps",
+                packages=({"path": ".", "type": "gomod"},),
+                check_files=["output.json", "fetch_deps_sha256sums.json"],
+                expected_rc=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            id="gomod_with_deps",
```

Comment:
It looks like we could get the pytest `id` at runtime and get rid of the duplicate `test_case` attribute https://stackoverflow.com/questions/56466111/how-to-find-the-test-id-for-parametrized-tests-during-execution-of-the-test
would that be worth it?

Reply:
I was briefly checking for something like that. Nice! :taco: